### PR TITLE
Overnight protocol LP adapter

### PR DESCRIPTION
**Pull request by matt5346**

### CODE REVIEW

**Comment by matt5346** [comment]
should be done, i did use Thegraph @nitish-91 instead of GoldSky, if its fine let me know pls

File: adapters/overnight/src/index_json_file.ts [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+  pool: string;
+  lpValue: string;
+}
+
+interface UserLPData {
+  totalLP: string;
+  pools: LPValueDetails[];
+}
+
+interface OutputData {
```
Comment (nitish-91) [inline adapters/overnight/src/index_json_file.ts:21]: Can you please make sure the output data follows the below schema mentioned in readme.
https://github.com/delta-hq/l2-lxp-liquidity-reward#output-data-schema

Comment (matt5346) [reply adapters/overnight/src/index_json_file.ts]: Now should be done!

Comment (matt5346) [reply adapters/overnight/src/index_json_file.ts]: is still smth wrong? @nitish-91

**Pull request merged**